Swinject/ServiceEntry.swift

```diff
@@ -8,19 +8,31 @@
 
 import Foundation
 
-/// The `ServiceEntryBase` class represents an entry of a registered service type.
-/// It is free from generics to store in a strongly typed collection.
-/// `ServiceEntry<Service>` should be actually used.
-public class ServiceEntryBase {
-    internal let factory: Any // Must be a function type.
+// A generic-type-free protocol to be the type of values in a strongly-typed collection.
+internal typealias ServiceEntryType = Any
+
+/// The `ServiceEntry<Service>` class represents an entry of a registered service type.
+/// As a returned instance from a `register` method of a `Container`, some configurations can be added.
```

Comment:
Length Violation: Line should be 100 characters or less: currently 103 characters